tests/sentry/mediators/external_issues/test_creator.py

```diff
@@ -0,0 +1,52 @@
+import responses
+
+from sentry.mediators.external_issues import Creator
+from sentry.models import PlatformExternalIssue
+from sentry.testutils import TestCase
+
+
+class TestCreator(TestCase):
+    def setUp(self):
+        super(TestCreator, self).setUp()
+
+        self.user = self.create_user(name="foo")
+        self.org = self.create_organization(owner=self.user)
+        self.project = self.create_project(slug="boop", organization=self.org)
+        self.group = self.create_group(project=self.project)
+
+        self.sentry_app = self.create_sentry_app(
+            name="foo", organization=self.org, webhook_url="https://example.com", scopes=()
+        )
+
+        self.install = self.create_sentry_app_installation(
+            slug="foo", organization=self.org, user=self.user
+        )
+
+    @responses.activate
+    def test_creates_platform_external_issue(self):
+        responses.add(
```

Comment:
we don't need to add responses here because unlike the `IssueLinkCreator` we aren't making any requests. All our lil mediator is doing is taking in the params needed to create the external issue instance on our end and return that newly made instance